dyngen/generate_dataset.R

```diff
@@ -0,0 +1,118 @@
+# Script created with the help of Robrecht Cannoodt
+library(tidyverse)
+library(dyngen)
+
+# I start off with few TFs, few targets, and 
+# no housekeeping genes
```

Comment:
The comment says no housekeeping but we have `num_hks` set to 15.  Is the comment stale?